tests/translator/output/error_graphqldatasource.json

```diff
@@ -1,3 +1,3 @@
 {
-  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 8. Resource with id [NoApiId] is invalid. Missing required property 'ApiId'. Resource with id [NoBaseTableTTL] is invalid. Property 'DynamoDBConfig.DeltaSyncConfig.BaseTableTTL' should be a string. Resource with id [NoConfigPropertyDefined] is invalid. 'DynamoDBConfig' must be defined when 'Type' is 'AMAZON_DYNAMODB'. Resource with id [NoDeltaSyncTableName] is invalid. Property 'DynamoDBConfig.DeltaSyncConfig.DeltaSyncTableName' should be a string. Resource with id [NoDeltaSyncTableTTL] is invalid. Property 'DynamoDBConfig.DeltaSyncConfig.DeltaSyncTableTTL' should be a string. Resource with id [NoDynamoDBTableName] is invalid. Property 'DynamoDBConfig.TableName' should be a string. Resource with id [NoType] is invalid. Missing required property 'Type'. Resource with id [UnknownType] is invalid. 'WEIRD_TYPE' is not a supported data source type."
```

Comment:
Can you add a test that adding some unknown property under `Logging` causes failure? 👉👈
So far I don't think we've failed, but this hopefully will (which is great).

Reply:
Added